src/main.ts

```diff
@@ -21,100 +21,100 @@ export async function run(
   getOctokit = github.getOctokit,
   git = simpleGit()
 ): Promise<void> {
-  try {
-    const inputs = parseInputs()
-    const options = createOptions(inputs)
+  const inputs = parseInputs()
+  const options = createOptions(inputs)
 
-    const [nodeMajorString] = (
-      process.version.startsWith('v')
-        ? process.version.slice(1)
-        : process.version
-    ).split('.')
-    const majorVersion = parseInt(nodeMajorString, 10)
-    const isUnsupportedVersion = majorVersion < 20
+  // https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/troubleshooting-workflows/enabling-debug-logging
+  if (
+    process.env['ACTIONS_RUNNER_DEBUG'] === 'true' ||
+    process.env['ACTIONS_STEP_DEBUG'] === 'true'
+  ) {
+    options.debug = true
+  }
```

Comment:
Is this really different to `core.isDebug()`? I see they use a slightly different enviroment variable (`RUNNER_DEBUG`) in the implementation: https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L233-L235

Either way, I would prefer this logic to live in `createOptions` function.

Reply:
It is either their docs that is wrong or some inconsistency but this was only way how it worked in all cases 😅 I'll move it